source/guide_mailman-3.rst

```diff
@@ -462,20 +462,78 @@ Updates
 =======
 As Mailman 3 consists of multiple independent projects, there is no single RSS feed. To check for updates, you can use ``pip`` on your uberspace:
 
-.. code :: bash
+::
 
  [isabell@stardust ~]$ pip3.8 list --outdated --user
  [isabell@stardust ~]$
 
 If there are outdated packages, update the mailman packages and their dependencies using:
 
-.. code :: bash
+::
 
  [isabell@stardust ~]$ pip3.8 install --user --upgrade mailman postorius hyperkitty mailman-hyperkitty whoosh uwsgi
  [isabell@stardust ~]$
 
 .. note:: Even after ``pip --upgrade``, there might be outdated packages. This is the case if mailman's dependencies demand a specific version, e.g. `Django<2.2,>=1.11`, and is nothing to worry about.
 
+Migration from Mailman 2
+========================
+If you followed the :lab:`Mailman 2 Lab Guide <guide_mailman.html>`, your Mailman 2 installation is located at ``/var/www/virtual/isabell/mailman``.
+To migrate your existing Mailman 2 installation to Mailman 3, first install Mailman 3 according to this guide. Then, log into the Postorius web interface on ``https://isabell.uber.space`` and create a new domain entry.
+
+Migrate lists
+-------------
+You need to migrate each one of your lists separately. First, create a new list ``https://isabell.uber.space/postorius/lists/new/`` with the same name as the original list (e.g. ``foo-list``). Don't worry, we will import everything in the next step.
+Import the list from Mailman 2 and build the archives using:
+
+::
+
+ [isabell@stardust ~]$ mailman import21 foo-list@isabell.uber.space /var/www/virtual/isabell/mailman/lists/foo-list/config.pck
+ [isabell@stardust ~]$ cd ~/mailman-suite/
+ [isabell@stardust mailman-suite]$ python3.8 manage.py hyperkitty_import -l foo-list@isabell.uber.space /var/www/virtual/isabell/mailman/archives/private/foo-list.mbox/foo-list.mbox
+ [isabell@stardust mailman-suite]$ python3.8 manage.py update_index_one_list foo-list@isabell.uber.space
+ [isabell@stardust ~]$ cd ~
+
+In case you opted for the qmail configuration via ``.qmail-default``, you can delete the old list-specific ``.qmail`` files:
+
+::
+
+ [isabell@stardust ~]$ rm -r .qmail-foo-list*
+
+Otherwise, update the ``.qmail`` files to use ``qmail-lmtp`` instead of ``vdeliver``:
+
+::
+
+ [isabell@stardust ~]$ echo "|/home/isabell/bin/qmail-lmtp 8024 1 isabell.local.uberspace.de" | tee ~/.qmail-foo-list{,-admin,-bounces,-confirm,-join,-leave,-owner,-request,-subscribe,-unsubscribe}
+
+
+Remove Mailman 2
+----------------
+First, remove the supervisord configuration and stop the service:
```

Comment:
acutally, the guide was changed: https://github.com/Uberspace/lab/commit/aa0d26dde4f8e0fde55ccaeb0dea10e03ce59580 to use cronjobs instead of the service, that should at least be mentioned here